src/main/java/pl/team/carrent/model/Rent.java

```diff
@@ -0,0 +1,144 @@
+package pl.team.carrent.model;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * @author: Maciej Kryger  [https://github.com/maciejkryger]
+ * @date : 27.10.2019 12:49
+ * *
+ * @className: Rent
+ * *
+ * *
+ ******************************************************/
+public class Rent {
+
+    @Id
+    @GeneratedValue(generator = "carRentSeq")
+    @SequenceGenerator(name = "carRentSeq", sequenceName = "car_rent_seq", allocationSize = 1)
+    private int id;
+
+    @ManyToOne(targetEntity = Car.class)
+    private Car car;
+
+    @ManyToOne(targetEntity = Client.class)
+    private Client client;
+
+    private int promotionId;
+
+    @ManyToOne(targetEntity = RentPoint.class)
+    private RentPoint rentPointStart;
```

Comment:
Brakuje miejsca, pracownika i daty oddania

Reply:
będzie to w RentHistory